site/app/templates/VerifyEmailForm.twig

```diff
@@ -0,0 +1,24 @@
+<div class="content shadow" id="login-box">
+    <div id="login-guest">
+     <!-- Add correct URL eventually -->
+        <h1> Verify your Email <a target=_blank href="https://submitty.org">
+        <i style="font-style:normal;" class="fa-question-circle"></i>
+        </a> </h1>
```

Comment:
Are you planning to change this in this PR?

Reply:
I will at least have the correct URL before this PR is merged, hopefully the docs will be soon after if not before this PR is merged.